loadtest/config.json

```diff
@@ -1,63 +1,71 @@
 {
```

Comment:
we should revert this. The one on the left is the correct one

Reply:
the main difference is the batch size + orders per block vs msgs per tx / txs per block. I believe the latter was refactored by @eric-zhu-uw when making loadtest changes. Do we not want to use that pattern anymore?